up_client_socket/cpp/conanfile.py

```diff
@@ -0,0 +1,64 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, replace_in_file, rm, rmdir
+import os
+
+class UpClientSocket(ConanFile):
+    name = "up_client_socket"
+    package_type = "library"
+    license = "Apache-2.0 license"
+    homepage = "https://github.com/eclipse-uprotocol"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "C++ up client socket library for testagent"
+    #topics = ("ulink client", "transport")
+    # Binary configuration
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
```

Comment:
double declaration, this can be safely removed

Reply:
done